packages/vm/src/modules/in_memory_cache.rs

```diff
@@ -0,0 +1,81 @@
+use clru::CLruCache;
+use wasmer_runtime_core::module::Module;
+
+use crate::{Checksum, Size, VmResult};
+
+const ESTIMATED_MODULE_SIZE: Size = Size::mebi(10);
+
+/// An in-memory module cache
+pub struct InMemoryCache {
+    lru: CLruCache<Checksum, Module>,
+}
+
+impl InMemoryCache {
+    /// Creates a new cache with the given size (in bytes)
+    pub fn new(size: Size) -> Self {
```

Comment:
I like this way of specifying size. However, I will need to pass this from Go to Rust. I guess I can just pass in a `usize`/`uint64` from Go and convert it in `cosmwasm-ffi`?

Reply:
Yeah, you can use `Size(bytes)` or `Size::mebi(limit_in_mb)` to construct